scripts/misc/download_hf_model.py

```diff
@@ -56,28 +129,33 @@
                 ignore_cert=args.ignore_cert,
             )
 
-            # A little hacky: run the Hugging Face download just to repair the symlinks in the HF cache file structure.
-            # This shouldn't actually download any files if the cache server download was successful, but should address
-            # a non-deterministic bug where the symlinks aren't repaired properly by the time the model is initialized.
-            log.info('Repairing Hugging Face cache symlinks')
+            if args.fallback == 'hf':
```

Comment:
@jerrychen109 is it okay to condition on this?

Reply:
should we condition on the save_dir being the HF cache? symlinks are just for the HF cache, right?